playbooks/ansible/roles/client.prep/tasks/centos.yml

```diff
@@ -12,6 +12,7 @@
       - podman
       - tox
       - sshfs
+      - python3-smbc
```

Comment:
Which repository provides `python3-smbc`?

Reply:
ahh sorry. Saw it on my Fedora desktop and assumed it would be available.

The problem here is that we need the python smbc module on the client to allow us to use the smbc python module. The problem is that there is no wheel available which means pip has to compile it. Which requires libsmbclient-devel installed on the system. How do we go about solving this issue without having to install libsmbclient-devel on the client?